tools/mo/openvino/tools/mo/front/tf/ObjectDetectionAPI.py

```diff
@@ -1521,8 +1521,18 @@ def ie_to_tf_proposals(graph: Graph, proposal: Node, match: SubgraphMatch, pipel
                                                                dict(name="reshape_swap_proposals_2d"), proposal)
         mark_input_as_in_correct_layout(proposal_reshape_2d, 0)
 
-        crop_and_resize_nodes_ids = [node_id for node_id in bfs_search(graph, [match.single_input_node(0)[0].id]) if
```

Comment:
it looks that `bfs_search` also finds the closest node in the topological order:

```
        for src_node, dst_node in graph.out_edges(cur_node_name):
```